src/theme/Feedback/index.tsx

```diff
@@ -0,0 +1,165 @@
+                        </svg>
+                        <h2 className="github-issue">Bug</h2>
+                    </button>
+                    <button
+                        className={`feedbackform-cta-btn ${category === 'idea' ? 'button-active' : ''}`}
+                        type="button"
+                        onClick={() => handleCategoryClick('idea')}
+                    >
+                        <svg width="18" height="25" viewBox="0 0 18 25" fill="none" xmlns="http://www.w3.org/2000/svg">
+                            <path fill-rule="evenodd" clip-rule="evenodd" d="M13.3605 17.5625C16.1274 16.0272 18 13.076 18 9.6875C18 4.71694 13.9706 0.6875 9 0.6875C4.02944 0.6875 0 4.71694 0 9.6875C0 13.076 1.87265 16.0272 4.63949 17.5625H13.3605Z" fill="currentColor"/>
+                            <rect x="4.5" y="18.6875" width="9" height="2.25" rx="0.5625" fill="currentColor"/>
+                            <path d="M11.25 22.0625C11.25 23.3051 10.2426 24.3125 9 24.3125C7.75736 24.3125 6.75 23.3051 6.75 22.0625C6.75 22.0625 10.0074 22.0643 11.25 22.0643V22.0625Z" fill="currentColor"/>
+                        </svg>
+                        <h2 className="github-issue">Idea</h2>
+                    </button>
+                    <button
+                        className={`feedbackform-cta-btn ${category === 'other' ? 'button-active' : ''}`}
+                        type="button"
+                        onClick={() => handleCategoryClick('other')}
+                    >
+                        <svg width="27" height="6" viewBox="0 0 27 6" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+                            <circle cx="3.5" cy="3" r="3" fill="currentColor"/>
+                            <circle cx="13.5" cy="3" r="3" fill="currentColor"/>
+                            <circle cx="23.5" cy="3" r="3" fill="currentColor"/>
+                        </svg>
+                        <h2 className="github-issue">Other</h2>
+                    </button>
+                    </div>
+                    <div className={`feedbackform-input-url ${inputFocus ? 'input-focus' : ''}`}>
+                        <input
+                            type="text"
+                            value={url}
+                            onChange={(e) => setUrl(e.target.value)}
```

Comment:
Please double-check if there's any issue with setURL here. It isn't picking up the URL for the page I'm on.
 
![image](https://github.com/surrealdb/docs.surrealdb.com/assets/5947670/01c7d2be-99e8-4320-9347-e3cae93f89fd)

Reply:
@naisofly It's not supposed to work like this, i'm afraid. A user should be able to type or copy/paste their own URL addresses on that input element, it's not just automatically wrapping the url page you are on. :)

Reply:
@naisofly the URL is picked up only when the person doesn't specify one. This is useful  for  when they're not on the page they want to give feedback on